src/client/browser/index.js

```diff
@@ -2,22 +2,28 @@
 // TODO: Get rid of Pinkie after dropping IE11
 import COMMAND from '../../browser/connection/command';
 import HeartbeatStatus from '../../browser/connection/heartbeat-status';
-import { UNSTABLE_NETWORK_MODE_HEADER } from '../../browser/connection/unstable-network-mode';
 import { HEARTBEAT_INTERVAL } from '../../utils/browser-connection-timeouts';
 
+/*eslint-disable no-restricted-properties*/
+const LOCATION_HREF   = document.location.href;
+const LOCATION_ORIGIN = document.location.origin;
+/*eslint-enable no-restricted-properties*/
+
+const STATUS_RETRY_DELAY = 1000;
+const MAX_STATUS_RETRY   = 5;
+
+const SERVICE_WORKER_LOCATION = LOCATION_ORIGIN + `/service-worker.js`;
```

Comment:
Move services routes to constant and use it here.

Reply:
I don't get it (
It's already in const, Do you want to extract `/service-worker.js` to the separate const?

Reply:
Same as here - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/proxy/service-routes.ts